packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -5,8 +5,15 @@
 <section>
   <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
   <p class="dummy-paragraph">
-    Tabs allow users to move between different views within the same context and at the same level of hierarchy, ie. UI
-    vs CLI, macOS vs Windows vs Linux, etc.
+    Tabs allow users to
+    <strong>move between different views within the same context</strong>
+    and at the same level of hierarchy, ie. UI vs. CLI, macOS vs. Windows vs. Linux, etc.
+  </p>
+  <p class="dummy-paragraph">
+    Tabs are not intended for primary navigation. For this reason, we chose to use HTML
+    <code class="dummy-code">button</code>
+    elements for activating individual tabs vs. links. If your use case includes a usage of a Tabs-like component for
+    primary navigation, consider reaching out to your designer and working with them to refactor the design if feasible.
   </p>
 </section>
 
```

Comment:
[**Related Reference**](https://www.unr.edu/writing-speaking-center/student-resources/writing-speaking-resources/british-american-english):
"American English puts periods after abbreviations.
Dr., Mr., Mrs., Ms., and vs. all have periods after them."